docs/resources/ui/components/built-in-components/flippablecard.md

```diff
@@ -0,0 +1,153 @@
+---
+slug: flippable-card
+title: Flippable Card
+tags: [Layout Elements, Components]
```

Comment:
Ensure this is under Layout Elements in widget builder.

Reply:
Yes, it is.
![Screenshot 2024-08-06 at 4 47 27 PM](https://github.com/user-attachments/assets/5dcb2e95-c587-4e6d-a680-3020e26e2827)